README.md

```diff
@@ -8,13 +8,38 @@
 
 ## Overview
 
-This repository contains the Liferay Screens source code, in addition to several example projects that illustrate how Screens is used. For detailed instructions on developing mobile apps with Screens, see the Screens documentation for [Android](https://dev.liferay.com/develop/tutorials/-/knowledge_base/6-2/android-apps-with-liferay-screens) and [iOS](https://dev.liferay.com/develop/tutorials/-/knowledge_base/6-2/ios-apps-with-liferay-screens) on the Liferay Developer Network (LDN). 
+This repository contains the Liferay Screens source code, in addition to several example projects that illustrate how Screens is used. For detailed instructions on developing mobile apps with Screens, see the Screens documentation for [Android](https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/android-apps-with-liferay-screens) and [iOS](https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/ios-apps-with-liferay-screens) on the Liferay Developer Network (LDN). 
 
 Brief descriptions of this repository's contents are listed here:
 
 - `android`: the Liferay Screens for Android codebase, View Sets, and example projects.
 - `ios`: the Liferay Screens for iOS codebase, Themes, and example projects.
 
+For now, the available screenlets are:
+
+* **Login Screenlet**: Signs users in to a Liferay instance.
```

Comment:
What do you think about something like this?

**Login Screenlet** [[iOS](https://dev.liferay.com/develop/reference/-/knowledge_base/7-0/loginscreenlet-for-android) / [Android](https://dev.liferay.com/develop/reference/-/knowledge_base/7-0/loginscreenlet-for-android)]: Signs users in to a Liferay instance.